www/_posts/2018-02-23-cordova-android-7.1.0.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Cordova Android 7.1.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Android 7.1.0` has been released! This release fixes various bugs releated to the previous `cordova-android@7.0.0` release.
+
+To upgrade:
+
+    npm install -g cordova
```

Comment:
I know this is jsut copy/paste from an earlier blog post, but why are this and the next line here? Installing CLI seems out of place here and folder name is pure speculation.

Reply:
It is just the default we have used for a long time. Haven't heard any complaints about it before